### ISSUE TITLE
Unable to update cni config: No networks found in /etc/cni/net.d/                   

### ISSUE THREAD

**Issue by azman0101** [kubernetes/kops#5321]
1. What `kops` version are you running? The command `kops version`, will display
 this information.
```kops version
Version 1.9.1 (git-ba77c9ca2)
```
2. What Kubernetes version are you running? `kubectl version` will print the
 version if a cluster is running or provide the Kubernetes version specified as
 a `kops` flag.

```
kubectl version
Client Version: version.Info{Major:"1", Minor:"9", GitVersion:"v1.9.6", GitCommit:"9f8ebd171479bec0ada837d7ee641dec2f8c6dd1", GitTreeState:"clean", BuildDate:"2018-03-21T15:21:50Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"9", GitVersion:"v1.9.6", GitCommit:"9f8ebd171479bec0ada837d7ee641dec2f8c6dd1", GitTreeState:"clean", BuildDate:"2018-03-21T15:13:31Z", GoVersion:"go1.9.3", Compiler:"gc", Platform:"linux/amd64"}
```
3. What cloud provider are you using?
AWS

4. What commands did you run?  What is the simplest way to reproduce this issue?

Instance kubelet fail to start correctly.

tested with `k8s-1.9-debian-jessie-amd64-hvm-ebs-2018-03-11` and `k8s-1.9-debian-stretch-amd64-hvm-ebs-2018-03-11`

5. What happened after the commands executed?

Instance keep unregistered and a restart of kubelet manually fix the problem.

6. What did you expect to happen?

Instance registered itself automatically

7. Please provide your cluster manifest. Execute
  `kops get --name my.example.com -o yaml` to display your cluster manifest.
  You may want to remove your cluster name and other sensitive information.

```
apiVersion: kops/v1alpha2
kind: Cluster
metadata:
  name: preprod-k8s.aws.mydomain.com
spec:
  api:
    loadBalancer:
      type: Public
  authorization:
    rbac: {}
  channel: stable
  cloudLabels:
    Environment: staging
    oxa-no-monitoring: "true"
  cloudProvider: aws
  configBase: s3://mycompany-kops-states-staging/preprod-k8s.aws.mydomain.com
  etcdClusters:
  - enableEtcdTLS: true
    etcdMembers:
    - instanceGroup: master-eu-central-1a
      name: a
    - instanceGroup: master-eu-central-1b
      name: b
    - instanceGroup: master-eu-central-1c
      name: c
    name: main
    version: 3.0.17
  - enableEtcdTLS: true
    etcdMembers:
    - instanceGroup: master-eu-central-1a
      name: a
    - instanceGroup: master-eu-central-1b
      name: b
    - instanceGroup: master-eu-central-1c
      name: c
    name: events
    version: 3.0.17
  iam:
    allowContainerRegistry: true
    legacy: false
  kubeAPIServer:
    runtimeConfig:
      autoscaling/v2beta1: "true"
  kubeControllerManager:
    horizontalPodAutoscalerUseRestClients: true
  kubelet:
    enableCustomMetrics: true
    kubeletCgroups: /systemd/system.slice
    runtimeCgroups: /systemd/system.slice
  kubernetesApiAccess:
  - XXX.XXX.XXX.XXX/YY
  - XXX.XXX.XXX.XXX/YY
  - XXX.XXX.XXX.XXX/YY
  - XXX.XXX.XXX.XXX/YY
  - XXX.XXX.XXX.XXX/YY
  kubernetesVersion: 1.9.6
  masterInternalName: api.internal.preprod-k8s.aws.mydomain.com
  masterKubelet:
    kubeletCgroups: /systemd/system.slice
    runtimeCgroups: /systemd/system.slice
  masterPublicName: api.preprod-k8s.aws.mydomain.com
  networkCIDR: XXX.XXX.XXX.XXX/YY
  networking:
    weave:
      mtu: 8912
  nonMasqueradeCIDR: XXX.XXX.XXX.XXX/YY
  sshAccess:
  - XXX.XXX.XXX.XXX/YY
  - XXX.XXX.XXX.XXX/YY
  subnets:
  - cidr: XXX.XXX.XXX.XXX/YY
    name: eu-central-1a
    type: Private
    zone: eu-central-1a
  - cidr: XXX.XXX.XXX.XXX/YY
    name: eu-central-1b
    type: Private
    zone: eu-central-1b
  - cidr: XXX.XXX.XXX.XXX/YY
    name: eu-central-1c
    type: Private
    zone: eu-central-1c
  - cidr: XXX.XXX.XXX.XXX/YY
    name: utility-eu-central-1a
    type: Utility
    zone: eu-central-1a
  - cidr: XXX.XXX.XXX.XXX/YY
    name: utility-eu-central-1b
    type: Utility
    zone: eu-central-1b
  - cidr: XXX.XXX.XXX.XXX/YY
    name: utility-eu-central-1c
    type: Utility
    zone: eu-central-1c
  topology:
    dns:
      type: Public
    masters: private
    nodes: private

---

apiVersion: kops/v1alpha2
kind: InstanceGroup
metadata:
  creationTimestamp: 2018-06-12T13:16:28Z
  labels:
    kops.k8s.io/cluster: preprod-k8s.aws.mydomain.com
  name: master-eu-central-1a
spec:
  image: kope.io/k8s-1.9-debian-jessie-amd64-hvm-ebs-2018-03-11
  machineType: m4.large
  maxSize: 1
  minSize: 1
  nodeLabels:
    kops.k8s.io/instancegroup: master-eu-central-1a
  role: Master
  rootVolumeSize: 64
  subnets:
  - eu-central-1a

---

apiVersion: kops/v1alpha2
kind: InstanceGroup
metadata:
  creationTimestamp: 2018-06-12T13:16:28Z
  labels:
    kops.k8s.io/cluster: preprod-k8s.aws.mydomain.com
  name: master-eu-central-1b
spec:
  image: kope.io/k8s-1.9-debian-jessie-amd64-hvm-ebs-2018-03-11
  machineType: m4.large
  maxSize: 1
  minSize: 1
  nodeLabels:
    kops.k8s.io/instancegroup: master-eu-central-1b
  role: Master
  rootVolumeSize: 64
  subnets:
  - eu-central-1b

---

apiVersion: kops/v1alpha2
kind: InstanceGroup
metadata:
  creationTimestamp: 2018-06-12T13:16:28Z
  labels:
    kops.k8s.io/cluster: preprod-k8s.aws.mydomain.com
  name: master-eu-central-1c
spec:
  image: kope.io/k8s-1.9-debian-jessie-amd64-hvm-ebs-2018-03-11
  machineType: m4.large
  maxSize: 1
  minSize: 1
  nodeLabels:
    kops.k8s.io/instancegroup: master-eu-central-1c
  role: Master
  rootVolumeSize: 64
  subnets:
  - eu-central-1c

---

apiVersion: kops/v1alpha2
kind: InstanceGroup
metadata:
  creationTimestamp: 2018-06-13T09:23:37Z
  labels:
    kops.k8s.io/cluster: preprod-k8s.aws.mydomain.com
  name: nodes-eu-central-1a
spec:
  cloudLabels:
    k8s.io/cluster-autoscaler/node-template/label/instance_type: balanced
    k8s.io/cluster-autoscaler/node-template/label/ondemand-node: "true"
    oxa-no-monitoring: "true"
  image: kope.io/k8s-1.9-debian-jessie-amd64-hvm-ebs-2018-03-11
  machineType: m4.2xlarge
  maxSize: 1
  minSize: 0
  nodeLabels:
    instance_type: balanced
    kops.k8s.io/instancegroup: nodes-eu-central-1a
    ondemand-node: "true"
  role: Node
  rootVolumeSize: 110
  subnets:
  - eu-central-1a

---

apiVersion: kops/v1alpha2
kind: InstanceGroup
metadata:
  creationTimestamp: 2018-06-13T09:23:38Z
  labels:
    kops.k8s.io/cluster: preprod-k8s.aws.mydomain.com
  name: nodes-eu-central-1b
spec:
  cloudLabels:
    k8s.io/cluster-autoscaler/node-template/label/instance_type: balanced
    k8s.io/cluster-autoscaler/node-template/label/ondemand-node: "true"
    oxa-no-monitoring: "true"
  image: kope.io/k8s-1.9-debian-jessie-amd64-hvm-ebs-2018-03-11
  machineType: m4.2xlarge
  maxSize: 1
  minSize: 0
  nodeLabels:
    instance_type: balanced
    kops.k8s.io/instancegroup: nodes-eu-central-1b
    ondemand-node: "true"
  role: Node
  rootVolumeSize: 110
  subnets:
  - eu-central-1b

---

apiVersion: kops/v1alpha2
kind: InstanceGroup
metadata:
  creationTimestamp: 2018-06-13T09:23:38Z
  labels:
    kops.k8s.io/cluster: preprod-k8s.aws.mydomain.com
  name: nodes-eu-central-1c
spec:
  cloudLabels:
    k8s.io/cluster-autoscaler/node-template/label/instance_type: balanced
    k8s.io/cluster-autoscaler/node-template/label/ondemand-node: "true"
    oxa-no-monitoring: "true"
  image: kope.io/k8s-1.9-debian-jessie-amd64-hvm-ebs-2018-03-11
  machineType: m4.2xlarge
  maxSize: 1
  minSize: 0
  nodeLabels:
    instance_type: balanced
    kops.k8s.io/instancegroup: nodes-eu-central-1c
    ondemand-node: "true"
  role: Node
  rootVolumeSize: 110
  subnets:
  - eu-central-1c

---

apiVersion: kops/v1alpha2
kind: InstanceGroup
metadata:
  creationTimestamp: 2018-06-13T11:00:06Z
  labels:
    kops.k8s.io/cluster: preprod-k8s.aws.mydomain.com
  name: nodes-spot-1a
spec:
  cloudLabels:
    k8s.io/cluster-autoscaler/node-template/label/instance_type: balanced
    k8s.io/cluster-autoscaler/node-template/label/spot-node: "true"
    oxa-no-monitoring: "true"
  image: kope.io/k8s-1.9-debian-jessie-amd64-hvm-ebs-2018-03-11
  machineType: m4.2xlarge
  maxPrice: "0.48"
  maxSize: 5
  minSize: 0
  nodeLabels:
    instance_type: balanced
    kops.k8s.io/instancegroup: nodes-spot-1a
    spot-node: "true"
  role: Node
  rootVolumeSize: 110
  rootVolumeType: gp2
  subnets:
  - eu-central-1a

---

apiVersion: kops/v1alpha2
kind: InstanceGroup
metadata:
  creationTimestamp: 2018-06-13T11:00:06Z
  labels:
    kops.k8s.io/cluster: preprod-k8s.aws.mydomain.com
  name: nodes-spot-1b
spec:
  cloudLabels:
    k8s.io/cluster-autoscaler/node-template/label/instance_type: balanced
    k8s.io/cluster-autoscaler/node-template/label/spot-node: "true"
    oxa-no-monitoring: "true"
  image: kope.io/k8s-1.9-debian-jessie-amd64-hvm-ebs-2018-03-11
  machineType: m4.2xlarge
  maxPrice: "0.48"
  maxSize: 5
  minSize: 0
  nodeLabels:
    instance_type: balanced
    kops.k8s.io/instancegroup: nodes-spot-1b
    spot-node: "true"
  role: Node
  rootVolumeSize: 110
  rootVolumeType: gp2
  subnets:
  - eu-central-1b

---

apiVersion: kops/v1alpha2
kind: InstanceGroup
metadata:
  creationTimestamp: 2018-06-13T11:00:06Z
  labels:
    kops.k8s.io/cluster: preprod-k8s.aws.mydomain.com
  name: nodes-spot-1c
spec:
  cloudLabels:
    k8s.io/cluster-autoscaler/node-template/label/instance_type: balanced
    k8s.io/cluster-autoscaler/node-template/label/spot-node: "true"
    oxa-no-monitoring: "true"
  image: kope.io/k8s-1.9-debian-jessie-amd64-hvm-ebs-2018-03-11
  machineType: m4.2xlarge
  maxPrice: "0.48"
  maxSize: 5
  minSize: 0
  nodeLabels:
    instance_type: balanced
    kops.k8s.io/instancegroup: nodes-spot-1c
    spot-node: "true"
  role: Node
  rootVolumeSize: 110
  rootVolumeType: gp2
  subnets:
  - eu-central-1c

---

apiVersion: kops/v1alpha2
kind: InstanceGroup
metadata:
  creationTimestamp: 2018-06-13T10:10:59Z
  labels:
    kops.k8s.io/cluster: preprod-k8s.aws.mydomain.com
  name: nodes-spot-cpu-1a
spec:
  cloudLabels:
    k8s.io/cluster-autoscaler/node-template/label/instance_type: compute
    k8s.io/cluster-autoscaler/node-template/label/spot-node: "true"
    oxa-no-monitoring: "true"
  image: kope.io/k8s-1.9-debian-stretch-amd64-hvm-ebs-2018-03-11
  machineType: c4.2xlarge
  maxPrice: "0.4540"
  maxSize: 5
  minSize: 1
  nodeLabels:
    instance_type: compute
    kops.k8s.io/instancegroup: nodes-spot-cpu-1a
    spot-node: "true"
  role: Node
  rootVolumeSize: 110
  rootVolumeType: gp2
  subnets:
  - eu-central-1a

---

apiVersion: kops/v1alpha2
kind: InstanceGroup
metadata:
  creationTimestamp: 2018-06-13T10:10:59Z
  labels:
    kops.k8s.io/cluster: preprod-k8s.aws.mydomain.com
  name: nodes-spot-cpu-1b
spec:
  cloudLabels:
    k8s.io/cluster-autoscaler/node-template/label/instance_type: compute
    k8s.io/cluster-autoscaler/node-template/label/spot-node: "true"
    oxa-no-monitoring: "true"
  image: kope.io/k8s-1.9-debian-stretch-amd64-hvm-ebs-2018-03-11
  machineType: c4.2xlarge
  maxPrice: "0.4540"
  maxSize: 5
  minSize: 1
  nodeLabels:
    instance_type: compute
    kops.k8s.io/instancegroup: nodes-spot-cpu-1b
    spot-node: "true"
  role: Node
  rootVolumeSize: 110
  rootVolumeType: gp2
  subnets:
  - eu-central-1b

---

apiVersion: kops/v1alpha2
kind: InstanceGroup
metadata:
  creationTimestamp: 2018-06-13T10:10:59Z
  labels:
    kops.k8s.io/cluster: preprod-k8s.aws.mydomain.com
  name: nodes-spot-cpu-1c
spec:
  cloudLabels:
    k8s.io/cluster-autoscaler/node-template/label/instance_type: compute
    k8s.io/cluster-autoscaler/node-template/label/spot-node: "true"
    oxa-no-monitoring: "true"
  image: kope.io/k8s-1.9-debian-stretch-amd64-hvm-ebs-2018-03-11
  machineType: c4.2xlarge
  maxPrice: "0.4540"
  maxSize: 5
  minSize: 1
  nodeLabels:
    instance_type: compute
    kops.k8s.io/instancegroup: nodes-spot-cpu-1c
    spot-node: "true"
  role: Node
  rootVolumeSize: 110
  rootVolumeType: gp2
  subnets:
  - eu-central-1c

---

apiVersion: kops/v1alpha2
kind: InstanceGroup
metadata:
  creationTimestamp: 2018-06-13T11:00:13Z
  labels:
    kops.k8s.io/cluster: preprod-k8s.aws.mydomain.com
  name: nodes-spot-memory-1a
spec:
  cloudLabels:
    k8s.io/cluster-autoscaler/node-template/label/instance_type: memory
    k8s.io/cluster-autoscaler/node-template/label/spot-node: "true"
    oxa-no-monitoring: "true"
  image: kope.io/k8s-1.9-debian-jessie-amd64-hvm-ebs-2018-03-11
  machineType: r4.xlarge
  maxPrice: "0.32"
  maxSize: 5
  minSize: 0
  nodeLabels:
    instance_type: memory
    kops.k8s.io/instancegroup: nodes-spot-memory-1a
    spot-node: "true"
  role: Node
  rootVolumeSize: 110
  rootVolumeType: gp2
  subnets:
  - eu-central-1a

---

apiVersion: kops/v1alpha2
kind: InstanceGroup
metadata:
  creationTimestamp: 2018-06-13T11:00:13Z
  labels:
    kops.k8s.io/cluster: preprod-k8s.aws.mydomain.com
  name: nodes-spot-memory-1b
spec:
  cloudLabels:
    k8s.io/cluster-autoscaler/node-template/label/instance_type: memory
    k8s.io/cluster-autoscaler/node-template/label/spot-node: "true"
    oxa-no-monitoring: "true"
  image: kope.io/k8s-1.9-debian-jessie-amd64-hvm-ebs-2018-03-11
  machineType: r4.xlarge
  maxPrice: "0.32"
  maxSize: 5
  minSize: 0
  nodeLabels:
    instance_type: memory
    kops.k8s.io/instancegroup: nodes-spot-memory-1b
    spot-node: "true"
  role: Node
  rootVolumeSize: 110
  rootVolumeType: gp2
  subnets:
  - eu-central-1b

---

apiVersion: kops/v1alpha2
kind: InstanceGroup
metadata:
  creationTimestamp: 2018-06-13T11:00:14Z
  labels:
    kops.k8s.io/cluster: preprod-k8s.aws.mydomain.com
  name: nodes-spot-memory-1c
spec:
  cloudLabels:
    k8s.io/cluster-autoscaler/node-template/label/instance_type: memory
    k8s.io/cluster-autoscaler/node-template/label/spot-node: "true"
    oxa-no-monitoring: "true"
  image: kope.io/k8s-1.9-debian-jessie-amd64-hvm-ebs-2018-03-11
  machineType: r4.xlarge
  maxPrice: "0.32"
  maxSize: 5
  minSize: 0
  nodeLabels:
    instance_type: memory
    kops.k8s.io/instancegroup: nodes-spot-memory-1c
    spot-node: "true"
  role: Node
  rootVolumeSize: 110
  rootVolumeType: gp2
  subnets:
  - eu-central-1c

---

apiVersion: kops/v1alpha2
kind: InstanceGroup
metadata:
  creationTimestamp: 2018-06-13T09:40:03Z
  labels:
    kops.k8s.io/cluster: preprod-k8s.aws.mydomain.com
  name: services
spec:
  cloudLabels:
    k8s.io/cluster-autoscaler/node-template/label/instance_type: balanced
    k8s.io/cluster-autoscaler/node-template/label/ondemand-node: "true"
    oxa-no-monitoring: "true"
  image: kope.io/k8s-1.9-debian-jessie-amd64-hvm-ebs-2018-03-11
  machineType: m4.xlarge
  maxSize: 3
  minSize: 3
  nodeLabels:
    instance_type: balanced
    kops.k8s.io/instancegroup: services
    ondemand-node: "true"
    scheduler.alpha.kubernetes.io/critical-pod: ""
  role: Node
  rootVolumeSize: 100
  rootVolumeType: gp2
  subnets:
  - eu-central-1a
  - eu-central-1b
  - eu-central-1c
  taints:
  - dedicated=services:NoSchedule
```
8. Please run the commands with most verbose logging by adding the `-v 10` flag.
  Paste the logs into this report, or in a gist and provide the gist link here.

```
Jun 13 17:27:28 ip-XXX-XXX-XXX-XXX kubelet[XXX-XXX-XXX-XXX]: WXXX-XXX-XXX-XXX 17:27:28.XXX-XXX-XXX-XXX    XXX-XXX-XXX-XXX cni.go:171] Unable to update cni config: No networks found in /etc/cni/net.d/                   
Jun 13 17:27:28 ip-XXX-XXX-XXX-XXX kubelet[XXX-XXX-XXX-XXX]: EXXX-XXX-XXX-XXX 17:27:28.XXX-XXX-XXX-XXX    XXX-XXX-XXX-XXX kubelet.go:XXX-XXX-XXX-XXX] Container runtime network not ready: NetworkReady=false reason:NetworkPluginNotReady message:docker: network plugin is not ready: cni config uninitialized
Jun 13 17:27:31 ip-XXX-XXX-XXX-XXX kubelet[XXX-XXX-XXX-XXX]: EXXX-XXX-XXX-XXX 17:27:31.XXX-XXX-XXX-XXX    XXX-XXX-XXX-XXX kubelet_node_status.go:383] Error updating node status, will retry: error getting node "ip-XXX-XXX-XXX-XXX.eu-central-1.compute.internal": nodes "ip-XXX-XXX-XXX-XXX.eu-central-1.compute.internal" not found                                                                     
Jun 13 17:27:31 ip-XXX-XXX-XXX-XXX kubelet[XXX-XXX-XXX-XXX]: EXXX-XXX-XXX-XXX 17:27:31.XXX-XXX-XXX-XXX    XXX-XXX-XXX-XXX kubelet_node_status.go:383] Error updating node status, will retry: error getting node "ip-XXX-XXX-XXX-XXX.eu-central-1.compute.internal": nodes "ip-XXX-XXX-XXX-XXX.eu-central-1.compute.internal" not found                                                                     
Jun 13 17:27:31 ip-XXX-XXX-XXX-XXX kubelet[XXX-XXX-XXX-XXX]: EXXX-XXX-XXX-XXX 17:27:31.XXX-XXX-XXX-XXX    XXX-XXX-XXX-XXX kubelet_node_status.go:383] Error updating node status, will retry: error getting node "ip-XXX-XXX-XXX-XXX.eu-central-1.compute.internal": nodes "ip-XXX-XXX-XXX-XXX.eu-central-1.compute.internal" not found                                                                     
Jun 13 17:27:31 ip-XXX-XXX-XXX-XXX kubelet[XXX-XXX-XXX-XXX]: EXXX-XXX-XXX-XXX 17:27:31.XXX-XXX-XXX-XXX    XXX-XXX-XXX-XXX kubelet_node_status.go:383] Error updating node status, will retry: error getting node "ip-XXX-XXX-XXX-XXX.eu-central-1.compute.internal": nodes "ip-XXX-XXX-XXX-XXX.eu-central-1.compute.internal" not found                                                                     
Jun 13 17:27:31 ip-XXX-XXX-XXX-XXX kubelet[XXX-XXX-XXX-XXX]: EXXX-XXX-XXX-XXX 17:27:31.XXX-XXX-XXX-XXX    XXX-XXX-XXX-XXX kubelet_node_status.go:383] Error updating node status, will retry: error getting node "ip-XXX-XXX-XXX-XXX.eu-central-1.compute.internal": nodes "ip-XXX-XXX-XXX-XXX.eu-central-1.compute.internal" not found                                                                     
Jun 13 17:27:31 ip-XXX-XXX-XXX-XXX kubelet[XXX-XXX-XXX-XXX]: EXXX-XXX-XXX-XXX 17:27:31.XXX-XXX-XXX-XXX    XXX-XXX-XXX-XXX kubelet_node_status.go:375] Unable to update node status: update node status exceeds retry count
Jun 13 17:27:33 ip-XXX-XXX-XXX-XXX kubelet[XXX-XXX-XXX-XXX]: WXXX-XXX-XXX-XXX 17:27:33.XXX-XXX-XXX-XXX    XXX-XXX-XXX-XXX cni.go:171] Unable to update cni config: No networks found in /etc/cni/net.d/                   
Jun 13 17:27:33 ip-XXX-XXX-XXX-XXX kubelet[XXX-XXX-XXX-XXX]: EXXX-XXX-XXX-XXX 17:27:33.XXX-XXX-XXX-XXX    XXX-XXX-XXX-XXX kubelet.go:XXX-XXX-XXX-XXX] Container runtime network not ready: NetworkReady=false reason:NetworkPluginNotReady message:docker: network plugin is not ready: cni config uninitialized
```

Currently, I've seen this bug on Spot instance but unlike before **I've no line of log like this one**
```
"KubernetesCluster" nor "kubernetes.io/cluster/..." not found; Kubernetes may behave unexpectedly
```

**Comment by azman0101** [comment]
I've just find theses lines

```
Jun XX XX:XX:XX ip-XX-XX-XX-XX kubelet[1953]: E0613 XX:XX:XX.XXXXXX    1953 tags.go:94] Tag "KubernetesCluster" nor "kubernetes.io/cluster/..." not found; Kubernetes may behave unexpectedly.
Jun XX XX:XX:XX ip-XX-XX-XX-XX kubelet[1953]: W0613 XX:XX:XX.XXXXXX    1953 tags.go:78] AWS cloud - no clusterID filtering applied for shared resources; do not run multiple clusters in this AZ.
Jun XX XX:XX:XX ip-XX-XX-XX-XX kubelet[1953]: I0613 XX:XX:XX.XXXXXX    1953 server.go:305] Successfully initialized cloud provider: "aws" from the config file: ""
Jun XX XX:XX:XX ip-XX-XX-XX-XX kubelet[1953]: I0613 XX:XX:XX.XXXXXX    1953 server.go:541] cloud provider determined current node name to be ip-XX-XXX-XXX-XXX.eu-central-1.compute.internal
Jun XX XX:XX:XX ip-XX-XX-XX-XX kubelet[1953]: I0613 XX:XX:XX.XXXXXX    1953 manager.go:151] cAdvisor running in container: "/sys/fs/cgroup/cpu,cpuacct/system.slice/kubelet.service"
```

**Comment by azman0101** [comment]
This issue seems related to kubernetes/kubernetes#57382 and fix in k8s 1.10 and cherry-picked for k8s 1.8 and 1.9 as stated in https://github.com/kubernetes/kubernetes/issues/57382#issuecomment-370676582.

Mentioned in k8s 1.9.6 changelog https://github.com/kubernetes/kubernetes/blob/release-1.9/CHANGELOG-1.9.md#other-notable-changes-1 

> On AWS kubelet returns an error when started under conditions that do not allow it to work (AWS has not yet tagged the instance). (#60125, @vainu-arto)

I'm running k8s v1.9.6 and the bug fix kubernetes/kubernetes#60125 seems not working.


By the way, I've added check-aws-tags to spot InstanceGroup spec and it seems to work
```
  hooks:
  - manifest: |
      Type=oneshot
      ExecStart=/usr/bin/docker run --net host --restart no azman0101/check-aws-tags:latest
      ExecStartPost=/bin/systemctl restart kubelet.service
    name: ensure-aws-tags.service
    requires:
    - docker.service
    roles:
    - Node
```

**Comment by azman0101** [comment]
Cherry-picked in 1.9.7 ... Closing this issue
